apps/dev-recruiters/src/components/modules/FormPage/signUpForm.tsx

```diff
@@ -101,11 +108,14 @@ export default function SignUpForm({
           role: 'title' as string, //  role: position.title as string,
           project: { id: '1', slug: 'projectSlug' }, //router.query.slug as string },
           skills: [{ skill: '' }],
+          isAgeOver18: false,
+          isTermsAgreed: false,
         }}
         onSubmit={(
           values: NewApplicant,
           { setSubmitting }: FormikHelpers<NewApplicant>
         ) => {
+          console.log(values);
```

Comment:
Do we need this log in production?

Reply:
No, we don't need this log message

Reply:
@chungthuang removed the console statement